### PR TITLE
refactor: add dom lib types on tsconfig

### PR DESCRIPTION
**Pull request by EdamAme-x**
a type error occurs when using the browser part in the **islands**.

### CODE REVIEW

File: templates/x-basic/tsconfig.json [hunk 1]
```diff
@@ -6,11 +6,12 @@
     "strict": true,
     "lib": [
       "ESNext",
+      "dom"
```
Comment (yusukebe) [inline templates/x-basic/tsconfig.json:9]: Shall we capitalize these?

**Comment by yusukebe** [comment]
Hi @EdamAme-x 

I've commented. Check it!

File: templates/x-basic/tsconfig.json (hunk 1 above)
Comment (EdamAme-x) [reply templates/x-basic/tsconfig.json]: thanks @yusukebe

**Comment by EdamAme-x** [comment]
thanks @yusukebe

**Comment by yusukebe** [comment]
@EdamAme-x 

Thanks for the super fast response. Merge now.

**Pull request merged**